src/main/steps/la-portal/upload-your-documents/content.ts

```diff
@@ -19,7 +19,6 @@ const en = () => {
     infoBullet4: 'Annex A (if available)',
     infoBullet5: "any previous adoption certificates or court orders for the child/child's siblings",
     infoBullet6: 'any additional evidence you feel is relevant such maintenance agreements or photo ID',
-    note1: 'You can upload multiple documents on this page. All certificates must be certified.',
```

Comment:
In Case Note 1 is deleted, its recommended to rename Note2 variable as note 1 and make subsequent changes in NJK file as well